Clarify further. (#3616)
modelica/ModelicaSpecification: chapters/stream.tex

  Connection equations with stream variables are generated in a model when using \lstinline!inStream! or \lstinline!actualStream!,
  see \cref{stream-operator-instream-and-connection-equations} and \cref{stream-operator-actualstream}.
\end{itemize}

> The variable(s) with \lstinline!stream! and \lstinline!flow! prefix must be directly in the connector (possibly inherited) and not in a sub-connector.

Reviewer: But a sub-connector may still be a stream connector, right?
```suggestion
The variable with \lstinline!flow! prefix must exist at the same level of the connector as the variable with  \lstinline!stream! prefix; i.e., they can each be directly declared in the connector or be inherited, while a stream variable in a sub-connector would result in a separate stream sub-connector.
```

Author: Yes, that is possible - but as noted that is a separate stream connector.

> These variable(s) with \lstinline!stream! and \lstinline!flow! prefix must be directly in the connector (possibly inherited) and not in a sub-connector.

Author: ```suggestion
The variable(s) with \lstinline!stream! and \lstinline!flow! prefix must be directly in the connector (possibly inherited) and not in a sub-connector.
```

> The variable with \lstinline!flow! prefix must exist at the same level of the connector as the variable with  \lstinline!stream! prefix; i.e., they can each be directly declared in the connector or be inherited, while a stream variable in a sub-connector would result in a separate stream sub-connector.

Reviewer: Excessive space (mu fault):
```suggestion
The variable with \lstinline!flow! prefix must exist at the same level of the connector as the variable with \lstinline!stream! prefix; i.e., they can each be directly declared in the connector or be inherited, while a stream variable in a sub-connector would result in a separate stream sub-connector.
```
